app/src/main/res/layout/cell_view.xml

```diff
@@ -1,17 +1,50 @@
 <?xml version="1.0" encoding="utf-8"?>
-<LinearLayout
-    xmlns:android="http://schemas.android.com/apk/res/android"
-    android:orientation="horizontal"
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
     android:layout_width="match_parent"
-    android:layout_height="wrap_content"
-    android:paddingTop="10dp"
-    android:paddingBottom="10dp"
-    >
+    android:layout_height="80dp"
+    tools:context=".TableAdapter">
 
     <TextView
-        android:id="@+id/text_view"
+        android:id="@+id/time_text_view"
         android:layout_width="0dp"
-        android:layout_height="wrap_content"
-        android:layout_weight="1"
+        android:layout_height="30dp"
+        android:textSize="22sp"
+        app:layout_constraintLeft_toLeftOf="parent"
+        app:layout_constraintRight_toRightOf="parent"
+        app:layout_constraintTop_toTopOf="parent"
         />
-</LinearLayout>
\ No newline at end of file
+
+    <TextView
```

Comment:
To make it easier to preview your layouts, you can use tools:text to set the text which is displayed in the layout previewer in Android Studio, which you might find useful 🙂